doc/markdown/user/Introduction.md

```diff
@@ -0,0 +1,326 @@
+
+The *alpaka* library is aimed at parallelization within nodes of a cluster.
+It does not compete with libraries for distribution of processes across nodes and communication among those.
+For these purposes libraries like MPI (Message Passing Interface) or others should be used.
+MPI is situated one layer higher and can be combined with *alpaka* to facilitate the hardware of a whole heterogeneous cluster.
+The *alpaka* library can be used for parallelization within nodes, MPI for parallelization across nodes.
+
+
+Comparison
+----------
+
+The following table summarizes which of the problems mentioned in section about the problems can be solved by current intra-node parallelization frameworks and the proof-of-concept *alpaka* abstraction library.
+
+| Framework / API | Open-Source | Free | Single-Source C++ | Portability | Heterogenity | Maintainability | Testability | Optimizability | Data structure agnostic |
+| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
+| CUDA			| :x:              	| :white_check_mark: | :white_check_mark: | :x:               | :x:               | :x:               | :x:               | :white_check_mark: | :white_check_mark: |
+| PGI CUDA-x86	| :x:             	| :x:               | :white_check_mark: | :white_check_mark: | :large_orange_diamond: | :white_check_mark: | :white_check_mark: | :x:               | :white_check_mark: |
+| GPU Ocelot		| :white_check_mark:	| :white_check_mark: | :white_check_mark: | :white_check_mark: | :large_orange_diamond: | :white_check_mark: | :white_check_mark: | :x:               | :white_check_mark: |
+| OpenMP			| :white_check_mark:	| :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :x:               | :white_check_mark: |
+| OpenACC			| :white_check_mark:	| :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :x:               | :white_check_mark: |
+| OpenCL			| :white_check_mark:	| :white_check_mark: | :x:               | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :x:               | :white_check_mark: |
+| SYCL			| :white_check_mark:	| (:ballot_box_with_check:) | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | (:ballot_box_with_check:) | :white_check_mark: |
+| C++AMP			| :white_check_mark:	| :white_check_mark: | :white_check_mark: | (:ballot_box_with_check:) | :white_check_mark: | :white_check_mark: | :white_check_mark: | :x:               | :white_check_mark: |
+| KOKKOS			| :white_check_mark:	| :white_check_mark: | :white_check_mark: | :white_check_mark: | :large_orange_diamond: | :white_check_mark: | :white_check_mark: | :x:               | :large_orange_diamond: |
```

Comment:
In KOKKOS it is also possible to select the target architecture as policy of the parallel_for or other entry points to a parallel algorithm. The default behaviour is selected at make time if no explicit target is selected.
The same is possible for memory places.